services/worker-manager/package.json

```diff
@@ -12,5 +12,8 @@
     "lint": "eslint src/*.js test/*.js",
     "fixlint": "eslint --fix src/*.js test/*.js",
     "test": "mocha test/*_test.js"
+  },
+  "dependencies": {
```

Comment:
All dependencies are at the top-level `package.json` now

Reply:
Done!